apps/common/views/mixins.py

```diff
@@ -45,23 +39,3 @@ def test_func(self):
             if not permission_class().has_permission(self.request, self):
                 return False
         return True
-
-
-class RecordViewLogMixin:
-    model: Model
-
-    def record_logs(self, ids, action, detail, model=None, **kwargs):
-        with translation.override('en'):
-            model = model or self.model
-            resource_type = model._meta.verbose_name
-            create_or_update_operate_log(
-                action, resource_type, force=True, **kwargs
-            )
-            activities = [
-                ActivityLog(
-                    resource_id=resource_id, type=ActivityChoices.operate_log,
-                    detail=detail, org_id=current_org.id,
-                )
-                for resource_id in ids
-            ]
-            ActivityLog.objects.bulk_create(activities)
```

Comment:
The code seems to have few minor differences.

1. The docstring of `record_logs` method in class `RecordViewLogMixin`, it should be 'model' instead of 'Model'. Currently, the docstring points at an incorrect model attribute.

2.`test_func` function is also different in terms of indentation.

Both other fields like `permissions_classes` etc. are similar between both versions of the file and there aren't any significant changes that would require major adjustments.

I've provided my response in English as requested for clarity.